src/pages/LendingTeams/TeamListing.vue

```diff
@@ -492,6 +451,7 @@ export default {
 		userIsTeamMember(teamId) {
 			return this.myTeams.some(team => team.id === teamId);
 		},
+		teamCategoryFriendlyName
```

Comment:
Looks like an empty method or does this actually work for imported method? I thought we generally added those methods to the data object for the component.

Reply:
Its imported from the utils file

Reply:
Interesting, I didn't realize this worked, since I've used the data object. Good to know!